src/bokeh/models/annotations/legends.pyi

```diff
@@ -10,7 +10,8 @@ from dataclasses import dataclass
 from typing import Literal, Sequence
 
 # Bokeh imports
-from ..._types import CoordinateLike, NullStringSpec, TextLike
+from ..._specs import NullStringSpec
+from ..._types import CoordinateLike, TextLike
```

Comment:
This and other cases weren't found in the original PR, because we skip type-checking in many modules, which also makes mypy skip checking associated `*.pyi` files (at least that's my understanding).